### PR TITLE
add cyclone sublime package

### PR DESCRIPTION
**Pull request by classicwuhao**
- [x] I'm the package's author and/or maintainer.
- [x] I have have read [the docs][1].
- [x] I have tagged a release with a [semver][2] version number.
- [x] My package repo has a description and a README describing what it's for and how to use it.
- [x] My package doesn't add context menu entries. *
- [x] My package doesn't add key bindings. **
- [x] Any commands are available via the command palette.
- [x] Preferences and keybindings (if any) are listed in the menu and the command palette, and open in split view.
- [x] If my package is a syntax it doesn't also add a color scheme. ***
- [x] I use [.gitattributes][3] to exclude files from the package: images, test files, sublime-project/workspace.

My package supports syntax highlight for Cyclone specification language.  https://classicwuhao.github.io/cyclone_tutorial/tutorial-content.html

There are no packages like it in Package Control.

[1]: https://packagecontrol.io/docs/submitting_a_package
[2]: https://semver.org
[3]: https://www.git-scm.com/docs/gitattributes#_export_ignore

### CODE REVIEW

**Comment by classicwuhao** [comment]
update repo based no the review

**Comment by braver** [comment]
Please have a look at the [Scope Naming guide](https://www.sublimetext.com/docs/scope_naming.html) and make sure the scopes specific to your syntax end the short name of your language (in this case `.cyclone`).

**Comment by braver** [comment]
You may also want to try to tag a release without all the additional files. They're not needed for manual installation (people can simply just drop your repo into the Packages dir) and it seems like it trips up the bot here.

**Comment by classicwuhao** [comment]
I don't understand why it always report the same error. I have already removed .sublime-package files. Only syntax file exists. Can anyone explain this to me? Thanks

**Comment by braver** [comment]
You need to tag a new release, package control only looks at the tags in your repo.

File: repository/c.json [hunk 1]
```diff
@@ -4852,6 +4852,17 @@
 				}
 			]
 		},
+		{
+			"name": "Cyclone SyntaxHighlighting",
```
Comment (braver) [inline repository/c.json:4856]: Language packages should just be named after the language, it does a lot more than just highlighting.

**Comment by braver** [comment]
> Please have a look at the [Scope Naming guide](https://www.sublimetext.com/docs/scope_naming.html) and make sure the scopes specific to your syntax end the short name of your language (in this case `.cyclone`).

Don’t forget about this one.

**Comment by braver** [comment]
Please let us know if you want to re-open this PR and continue the review.

**Pull request closed without merging**